package.json

```diff
@@ -36,7 +36,7 @@
         "@codemirror/legacy-modes": "^6.3.1",
         "@emotion/react": "^11.10.5",
         "@emotion/styled": "^11.10.5",
-        "@exabyte-io/wave.js": "2023.4.4-0",
+        "@exabyte-io/wave.js": "git+https://github.com/Exabyte-io/wave.js#update/SOF-6665",
```

Comment:
For this to be converted to the published version, we need to fix tests in wave.js

Reply:
This just got published in https://github.com/Exabyte-io/wave.js/actions/runs/5448281141/jobs/9911409006. We need to update package-lock as well.